### PR TITLE
Docs typo fix. broadcast_refreshes_to method does not exist

### PR DESCRIPTION
**Pull request by ConfusedVorlon**
broadcasts_refreshes_to method _does_ exist

### CODE REVIEW

**Review by seanpdoyle — APPROVED** [review]

**Comment by dapi** [comment]
Thank for the issue.

`broadcast_refreshes` does not exist also ;)

```
$ ag broadcast_refreshes 
app/models/concerns/turbo/broadcastable.rb
90:# The +broadcast_refreshes+ class method configures the model to broadcast a "page refresh" on creates, 
94:#     broadcast_refreshes
107:# You can also specify the streamable declaratively by passing a symbol to the +broadcast_refreshes_to+ method:
111:#     broadcast_refreshes_to :board
```

**Comment by ConfusedVorlon** [comment]
I see an approval - but also the workflow requires an approval from a maintainer.

Can someone do that?

**Comment by jorgemanrubia** [comment]
Thanks @ConfusedVorlon. This was addressed by https://github.com/hotwired/turbo-rails/pull/608.

**Pull request closed without merging**
